test/functional/beacon_controller_test.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+require_relative '../test_helper'
+# let's see how my testing skills go…
+class BeaconControllerTest < ActionController::TestCase
+  context "Beacon" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.